packages/amazonq/src/lsp/chat/messages.ts

```diff
@@ -147,20 +162,21 @@ export function registerMessageListeners(
                 break
             }
             case chatRequestType.method: {
+                const chatParams = { ...message.params } as ChatParams
                 const partialResultToken = uuidv4()
                 const chatDisposable = languageClient.onProgress(chatRequestType, partialResultToken, (partialResult) =>
-                    handlePartialResult<ChatResult>(partialResult, encryptionKey, provider, message.params.tabId)
+                    handlePartialResult<ChatResult>(partialResult, encryptionKey, provider, chatParams.tabId)
                 )
 
                 const editor =
                     vscode.window.activeTextEditor ||
                     vscode.window.visibleTextEditors.find((editor) => editor.document.languageId !== 'Log')
                 if (editor) {
-                    message.params.cursorPosition = [editor.selection.active]
```

Comment:
I guess this means the demo vscode client from flare also has this problem, unless they fixed it after I copied this chunk of code

Reply:
Good to know, I'll check on that.